flask_static_digest/__init__.py

```diff
@@ -46,8 +46,10 @@ def _load_manifest(self, app):
         return manifest_dict
 
     def _prepend_host_url(self, host, filename):
-        return urljoin(self.host_url,
-                       "/".join([self.static_url_path, filename]))
+        parts = list( # removing empty strings
```

Comment:
Any chance we can write rewrite this and the line below it to be a more clear on what it does?